__init__.py

```diff
@@ -153,8 +153,26 @@ def all(self, args):
     df, df2, extracted_data = main_backend(args)
 
     main_frontend(extracted_data, args)
-    
+
+    print(df.columns)
+    print(df2.columns)
+    print(extracted_data.dtype)
     df2['SubmitDate'] = df2['SubmitDatetimeX'].dt.date
-    numerical_columns = df2.select_dtypes(include='number').columns
-    result = df2.groupby('SubmitDate')[['carbonFootprint', 'carbonFootprint_failedJobs', 'cost', 'cost_memoryNeededOnly', 'cost_failedJobs']].sum().reset_index()
-    result.to_csv(f"{args.userCWD}/data.csv", index=False)
\ No newline at end of file
+
+    result_submitdate = df2.groupby('SubmitDate').agg({
```

Comment:
I would again create a groupby_dict to avoid duplicating the below specification